assets/tests/components/notificationModals/chelseaRaisedNotificationModal.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { mount } from "enzyme"
+import ChelseaRaisedNotificationModal from "../../../src/components/notificationModals/chelseaRaisedNotificationModal"
+import { Notification, NotificationState } from "../../../src/realtime.d"
+
+describe("ChelseaRaisedNotificationModal", () => {
```

Comment:
**issue:** should also add a test for the close button interaction.

Reply:
fixed in 1213e1a